installation/install.sh

```
@@ -1799,9 +1799,15 @@ iz_install_key_manager(){
     trap 'traplogger $LINENO "$BASH_COMMAND" $?'  DEBUG
     if [ ! -z "$ZSTACK_YUM_REPOS" ]; then
         yum --disablerepo="*" --enablerepo="$ZSTACK_YUM_REPOS" clean metadata >/dev/null 2>&1
+        yum --disablerepo="*" --enablerepo="$ZSTACK_YUM_REPOS" list key-manager >/dev/null 2>&1
+        if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)